Simpson_Assign1/Program.cs

```diff
@@ -16,11 +16,131 @@ static void Main(string[] args)
+                        foreach (Student eachStudent in Students)
+                        {
+                            Console.WriteLine(eachStudent.ToString());
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("2"):
+                        Console.Write("Which major list would you like printed?");
+                        string major = Console.ReadLine();
+                        Console.Write("\nStudent List <{0} Majors>", major);
+                        Console.WriteLine("\n------------------------------------------------------");
+                        List<Student> foundStudent = Students.FindAll(x => x.Major == major);
+                        foreach (Student eachStudent in foundStudent)
+                        {
+                            Console.WriteLine(eachStudent.ToString());
+                            count++;
+                        }
+                        if (count == 0)
+                        {
+                            Console.WriteLine("There doesn't appear to be any students majoring in '{0}'.", major);
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("3"):
+                        Console.WriteLine("Which academic year would you like printed?");
+                        Console.Write("<Freshman, Sophomore, Junior, Senior, PostBacc> ");
+                        string year = Console.ReadLine();
+                        Console.Write("\nStudent List <{0} Majors>", year);
+                        Console.WriteLine("\n------------------------------------------------------");
+                        AcademicYear year2;
+                        Enum.TryParse(year, out year2);
+                        List<Student> foundStudent2 = Students.FindAll(x => x.Year == year2);
+                        foreach (Student eachStudent in foundStudent2)
+                        {
+                            Console.WriteLine(eachStudent.ToString());
+                            count++;
+                        }
+                        if (count == 0)
+                        {
+                            Console.WriteLine("There doesn't appear to be any students in '{0}'.", year);
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("4"):
+                        Console.WriteLine("\nCourse List <All Courses>:");
+                        Console.WriteLine("------------------------------------------------------");
+                        foreach (Course eachCourse in Courses)
+                        {
+                            Console.WriteLine(eachCourse.ToString());
+                        }
+                        Console.WriteLine("");
+                        break;
+                    case ("5"):
```

Comment:
All of the logic that checks whether a student is on the roster is already in the PrintRoster() method in the course class.  So after you find the appropriate course you can just do foundCourse.PrintRoster(Students);

This will pass all of the Students into the method and PrintRoster() will handle the rest.